kratos/sources/kratos_application.cpp

```diff
@@ -207,6 +207,11 @@ void KratosApplication::RegisterKratosCore() {
     KRATOS_REGISTER_GEOMETRY("Hexahedra3D8", mHexahedra3D8Prototype);
     KRATOS_REGISTER_GEOMETRY("Hexahedra3D20", mHexahedra3D20Prototype);
     KRATOS_REGISTER_GEOMETRY("Hexahedra3D27", mHexahedra3D27Prototype);
+    KRATOS_REGISTER_GEOMETRY("QuadraturePointGeometry1DinWorkingSpace1D", mQuadraturePointGeometry1DinWorkingSpace1D);
+    KRATOS_REGISTER_GEOMETRY("QuadraturePointGeometry1DinWorkingSpace2D", mQuadraturePointGeometry1DinWorkingSpace2D);
```

Comment:
Is this needed? When you add a geometry like this it produces pretty much none sense.

Reply:
The serializer requires the object to be registered when loading the object. This is not used to create the object, only to let the serializer know how the object looks like.

Reply:
Ok. Alright, then!

Reply:
Can you name them accordingly to their dimension:
Curve2D, Point 2D, Surface2D...

Reply:
shouldn't it then be Point1D, Point2D, Point3D, Surface2D, Surface3D, Solid3D?
And entire name like this: QuadraturePointGeometryPoint1D etc.?

Reply:
Exactly, that's what I had in mind. I think it's more understandable then 2D in ...